src/setupProxy.js

```diff
@@ -6,6 +6,7 @@ module.exports = (app) => {
     createProxyMiddleware({
       target: 'http://localhost:4000',
       changeOrigin: true,
+      ws: true,
```

Comment:
Jeebus, is that all it needed?  Even though the middleware shouldn't apply to the `/user` route (I assumed, based on that array)?